spikes/lk_touch_sensor_kit/main.cpp

```diff
@@ -0,0 +1,224 @@
+			break;
+		case Position::belt_left_front:
+			log_info("Belt left front is touched");
+			break;
+		case Position::belt_left_back:
+			log_info("Belt left back is touched");
+			break;
+		case Position::belt_right_front:
+			log_info("Belt right front is touched");
+			break;
+		case Position::belt_right_back:
+			log_info("Belt right back is touched");
+			break;
+		default:
+			break;
+	}
+	log_info("\n\n");
+	rtos::ThisThread::sleep_for(1s);
+}
+
+void printSensorReleased(Position position)
+{
+	switch (position) {
+		case Position::ear_left:
+			log_info("Ear left is released");
+			break;
+		case Position::ear_right:
+			log_info("Ear right is released");
+			break;
+		case Position::belt_left_front:
+			log_info("Belt left front is released");
+			break;
+		case Position::belt_left_back:
+			log_info("Belt left back is released");
+			break;
+		case Position::belt_right_front:
+			log_info("Belt right front is released");
+			break;
+		case Position::belt_right_back:
+			log_info("Belt right back is released");
+			break;
+	}
+	log_info("\n\n");
+	rtos::ThisThread::sleep_for(1s);
+}
+
+void calibration()
+{
+	TouchLed::turnOffAllLeds();
+
+	log_info("Put your hand on ear left sensor");
+	touch_sensor_kit.calibrate(Position::ear_left);
+	setColorEar(TouchLed::ears.at(1), RGB::pure_red);
+
+	log_info("Put your hand on ear right sensor");
+	touch_sensor_kit.calibrate(Position::ear_right);
+	setColorEar(TouchLed::ears.at(0), RGB::pure_red);
+
+	log_info("Put your hand on belt left front sensor");
+	touch_sensor_kit.calibrate(Position::belt_left_front);
+	setColorBelt(TouchLed::belt.at(0), RGB::pure_red);
+
+	log_info("Put your hand on belt left back sensor");
+	touch_sensor_kit.calibrate(Position::belt_left_back);
+	setColorBelt(TouchLed::belt.at(1), RGB::pure_red);
+
+	log_info("Put your hand on belt right front sensor");
+	touch_sensor_kit.calibrate(Position::belt_right_front);
+	setColorBelt(TouchLed::belt.at(2), RGB::pure_red);
+
+	log_info("Put your hand on belt right back sensor");
+	touch_sensor_kit.calibrate(Position::belt_right_back);
+	setColorBelt(TouchLed::belt.at(3), RGB::pure_red);
+
+	TouchLed::turnOffAllLeds();
+}
+
+auto main() -> int
+{
+	logger::init();
+
+	HelloWorld hello;
+	hello.start();
+
+	log_info("Hello, World!\n\n");
+
+	auto start = rtos::Kernel::Clock::now();
+
+	rtos::ThisThread::sleep_for(2s);
+
+	auto t = rtos::Kernel::Clock::now() - start;
+	log_info("A message from your board %s --> \"%s\" at %i s\n", MBED_CONF_APP_TARGET_NAME, hello.world,
+			 int(t.count() / 1000));
+
+	rtos::ThisThread::sleep_for(2s);
+
+	TouchLed::turnOffAllLeds();
+
+	touch_sensor_kit.init();
+	calibration();
+
+	auto turnOnLeds_func  = [&](Position position) { TouchLed::turnOnLeds(position); };
+	auto turnOffLeds_func = [&](Position position) { TouchLed::turnOffLeds(position); };
+
+	touch_sensor_kit.registerOnSensorTouched(turnOnLeds_func);
+	touch_sensor_kit.registerOnSensorReleased(turnOffLeds_func);
```

Comment:
Avoir ici, dans le meilleur des cas bien sûr, 

```suggestion
	touch_sensor_kit.registerOnSensorTouched(TouchLed::turnOnLeds);
	touch_sensor_kit.registerOnSensorReleased(TouchLed::turnOffLeds);
```

ou

```suggestion
	touch_sensor_kit.registerOnSensorTouched(printSensorTouched);
	touch_sensor_kit.registerOnSensorReleased(printSensorReleased);
```